docs/onboard-vendors-to-techpass.md

```diff
@@ -1,56 +1,42 @@
->1. While registering, ensure to provide the correct VPN ID.
->1. It might take up to two weeks for us to verify the VPN ID before sending the TechPass invite email. If you do not receive this email after two weeks, check if it is the same email address you provided while registering for TechPass or check your Spam, Junk Email, Deleted Items or Archive folder.
+> **Note**:
+> If you are vendor who has an email address in the format of *_from.XX@XX.gov.sg*, refer to [Onboard public officers](onboard-public-officers-using-non-se-machines) and follow the instructions to get your TechPass account and SEED.
 
-<div class="warn">
-<ul>
-<li>If you are a GCC 1.0 user and activating your TechPass account based on the below instructions, skip step 1.</li>
-<li>If you are a GCC Common Services vendor user with *_from@*.gov.sg - Please raise a <a href="https://go.gov.sg/techpass-sr">service request</a> to provision your accounts. You will need to provide a valid vendor company email address, mobile number, first name, last name, company and department. </li>
-<li>Other users need to complete the below steps using a non-SE GSIB machine.</li>
-<li>If you've started the TechPass onboarding process, make sure to complete it within the same session.</li>
-</ul>
-</div>
 
-*To get a TechPass account* :
+If you are a vendor, complete the following steps to get a TechPass account and [SEED](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/) provisioning. You need SEED provisioning to access SGTS services from your internet(non-GSIB) device.
+
+
+> **Tip**: Click the triangle to view the instructions to complete each step.
 
 ## Step 1. Get a TechPass account and request for SEED provision
 
-<details> <summary style="font-size:18px"> How to get a TechPass account and request for SEED provisioning?</summary>
+<details> <summary style="font-size:20px;font-weight:bold"> How to get a TechPass account and request for SEED provisioning?</summary>
 
- <div class="warn">
- <ul>
- <li>SEED provisioning is required only if you use your <b>non-GSIB device</b> to access SGTS services.</li>
- <li> Emails with <b>*_from@*.gov.sg</b> format are <b>not permitted</b>. You need to provide the company email ID supplied by the vendor company.</li>
- <li> If you already have an active TechPass account and need SEED provisioning, create a [service request](https://go.gov.sg/techpass-sr).</li>
- <li>If you are a public officer, refer to <a href="https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-public-officers-using-non-se-machines">Onboarding Public Officers to TechPass</a>.</li>
- </ul>
- </div>
+1. Contact your project manager or reporting officer to request for TechPass account and SEED provisioning.
 
+2. Please provide the required details in this request, such as your official vendor company email address, handphone number and project name.
```

Comment:
just an observation:
- sometimes i see handphone number and sometimes mobile number
- sometimes i see handphone and sometimes mobile phone

Reply:
Good point! let me check what the standard is in UK English. Changed them all to "mobile phone".